tilings/strategies/verification.py

```diff
@@ -32,10 +38,33 @@
 ]
 
 
-BasicVerificationStrategy = AtomStrategy
 TileScopeVerificationStrategy = VerificationStrategy[Tiling, GriddedPerm]
 
 
+class BasicVerificationStrategy(AtomStrategy):
+    @staticmethod
+    def count_objects_of_size(
+        comb_class: CombinatorialClass, n: int, **parameters: int
+    ) -> int:
+        """
+        Verification strategies must contain a method to count the objects.
+        """
+        if not isinstance(comb_class, Tiling):
+            raise NotImplementedError
+        cast(Tiling, comb_class)
+        gp = next(comb_class.minimal_gridded_perms())
+        expected = {"n": len(gp)}
+        for assumption in comb_class.assumptions:
+            count = len(
```

Comment:
I think this is a method we want on TrackingAssumption. It feels like we'll need to do that often

Reply:
Agree, I'll move it there.